csharp-selenium-webdriver-sample/SampleTest.cs

```diff
@@ -6,57 +6,72 @@
 using System;
 using System.IO;
 using Selenium.Axe;
-using OpenQA.Selenium.Chrome;
 
 namespace CSharpSeleniumWebdriverSample.Test
 {
     [TestClass]
-    [DeploymentItem("SamplePage.html")]
+    //  Install chromedriver to enable Chrome broswer and copy it to the same directory as the deployed test assemblies.
     [DeploymentItem("chromedriver.exe")]
+    //  Install geckodriver to enable Firefox brower and copy it to the same directory as the deployed test assemblies.
     [DeploymentItem("geckodriver.exe")]
     public class SampleTest
     {
+        //  Intance to control the browser.
         private IWebDriver _webDriver;
 
         [TestCleanup]
         public virtual void TearDown()
         {
+            //  You should quit the driver on TearDown which close every associated window.
             _webDriver?.Quit();
         }
 
 
         [TestMethod]
+        // Test on Chrome
```

Comment:
I don't think this comment is adding any value; it's pretty apparent that the code is intended to enable testing on Chrome. I think a more valuable comment here might be a 1-sentence summary of how the `DataRow` attribute works with a link to MSTest docs explaining it in more detail, or maybe suggestions for ways besides `DataRow` to achieve the "write test once, run multiple times on different browsers" behavior